ksa_compliance/ksa_compliance/doctype/zatca_phase_1_business_settings/zatca_phase_1_business_settings.json

```diff
@@ -0,0 +1,80 @@
+{
+ "actions": [],
+ "allow_rename": 1,
+ "creation": "2024-05-23 15:10:35.449147",
+ "doctype": "DocType",
+ "engine": "InnoDB",
+ "field_order": [
+  "company",
+  "vat_registration_number",
+  "type_of_transaction",
+  "column_break_gigi",
+  "address",
+  "status"
+ ],
+ "fields": [
+  {
+   "fieldname": "company",
+   "fieldtype": "Link",
+   "in_list_view": 1,
+   "label": "Company",
+   "options": "Company",
+   "reqd": 1,
+   "unique": 1
+  },
+  {
+   "fieldname": "vat_registration_number",
+   "fieldtype": "Data",
+   "label": "VAT Registration Number"
+  },
+  {
+   "fieldname": "type_of_transaction",
+   "fieldtype": "Select",
+   "label": "Type Of Transaction",
+   "options": "Simplified\nStandard Tax Invoice\nBoth"
+  },
+  {
+   "fieldname": "column_break_gigi",
+   "fieldtype": "Column Break"
+  },
+  {
+   "fieldname": "address",
+   "fieldtype": "Link",
+   "in_list_view": 1,
+   "label": "Address",
+   "link_filters": "[[\"Address\",\"link_name\",\"=\",\"eval:doc.company\"]]",
```

Comment:
I don't see anything that sets the value to the company primary address by default (2nd point in the [user story](https://openproject.lavaloon.com/projects/zatca-project/work_packages/3785/activity?query_id=172)).

Not sure if something like `eval: frappe.get_value('Address', {'company': doc.company, 'is_primary_address': 1})` would work in the default value field. If it doesn't, we can add a change listener on the 'company' field in JavaScript. In Python, we'd use the `validate` or `before_save` hooks.